src/elements/content-sidebar/__tests__/BoxAISidebar.test.tsx

```diff
@@ -3,14 +3,19 @@ import { userEvent } from '@testing-library/user-event';
 import { render, screen } from '../../../test-utils/testing-library';
 import BoxAISidebar, { BoxAISidebarProps } from '../BoxAISidebar';
 
-jest.mock('@box/box-ai-agent-selector', () => ({
-    ...jest.requireActual('@box/box-ai-agent-selector'),
-    BoxAiAgentSelectorWithApi: () => <div data-testid="sidebar-agent-selector" />,
-}));
+let MockBoxAiAgentSelectorWithApi: jest.Mock;
+jest.mock('@box/box-ai-agent-selector', () => {
+    MockBoxAiAgentSelectorWithApi = jest.fn();
+    return {
+        ...jest.requireActual('@box/box-ai-agent-selector'),
+        BoxAiAgentSelectorWithApi: MockBoxAiAgentSelectorWithApi,
+    };
+});
 
 const mockOnClearAction = jest.fn();
 jest.mock('@box/box-ai-content-answers', () => ({
     ...jest.requireActual('@box/box-ai-content-answers'),
+    // BoxAiContentAnswers: jest.fn().mockImplementation(() => <div data-testid="content-answers" />),
     withApiWrapper: Component => props => (
         <Component
```

Comment:
this component should just spread the props instead of adding each specific prop